src/wp-admin/includes/plugin.php

```diff
@@ -2473,3 +2473,87 @@ function paused_plugins_notice() {
 		__( 'Go to the Plugins screen' )
 	);
 }
+
+/**
+ * Renders an admin notice in case some plugins deactivated.
```

Comment:
Feels like a word is missing in this sentence.

- "some plugins are|were|have been deactivated."?
- "Renders an admin notice when a plugin was deactivated during an update"